nestjs-counter/src/temporal/workflows.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  condition,
+  defineQuery,
+  defineSignal,
+  proxyActivities,
+  setHandler,
+} from '@temporalio/workflow';
+import { IActivities } from './activities';
+
+export const incrementSignal = defineSignal<[number]>('increment');
+export const getValueQuery = defineQuery<number>('getValue');
+
+const { persist } = proxyActivities<IActivities>({
```

Comment:
I’d use the class..

Reply:
Fixed, I guess I just needed to upgrade SDK versions. With SDK v1.0.0-rc0, using the class led to TypeScript complaining "index signature for type 'string' is missing in type"